contracts/token/ERC20/ERC20Detailed.sol

```diff
@@ -10,34 +10,13 @@ import "./IERC20.sol";
  * just as on Ethereum all the operations are done in wei.
  */
 contract ERC20Detailed is IERC20 {
-  string private name_;
-  string private symbol_;
-  uint8 private decimals_;
+  string public name;
+  string public symbol;
+  uint8 public decimals;
```

Comment:
hey, what happened here? I suppose this was a failed merge.

Reply:
I think this needs to be merged into the new master again? not sure

Reply:
We must've merged this file wrong.